roles/uperf/templates/workload.yml.j2

```diff
@@ -148,6 +148,7 @@ spec:
                  -v \
 {% endif %}
                  ;
+                      exit $command_status;
```

Comment:
@vishnuchalla this may have to be reverted, ptal

Reply:
Done